flocker/node/agents/testtools/_cinder.py

```diff
@@ -3,14 +3,19 @@
 """
 Test helpers for ``flocker.node.agents.cinder``.
 """
-
 from zope.interface.verify import verifyObject
 
 from flocker.testtools import TestCase
 from ..cinder import (
     ICinderVolumeManager,
     INovaVolumeManager,
 )
+from ._blockdevice import require_backend
+
+
+# Raise SkipTest unless the functional test configuration has an openstack
+# backend.
+require_backend_openstack = require_backend('openstack')
```

Comment:
I did this to avoid writing `require_backend('openstack')` in lots of places across two test modules...but on reflection it's probably not worth the indirection. I'll remove it.